salma-nyagaka/ToBeNamed
# #165217462 Setup frontend testing

#### What does this PR do?
- Configure jest for unit testing
- Configure cypress for end to end testing

#### Description of Task to be completed?
- add jest and cypress for testing

#### How should this be manually tested
- Clone this repository
- Set up the project(check README)
- Checkout to this branch
- Run bundle
- Run the rails server using rails s
- Run `yarn run cypress open` for end to end testing
- Run `npm test`. 

#### Any background context you want to provide?
N/A

#### What are the relevant pivotal tracker stories?
#165217462(https://www.pivotaltracker.com/story/show/165217462)
